app/src/main/java/com/lucasfagundes/android_pokedex/view/MainActivity.kt

```diff
@@ -2,23 +2,29 @@ package com.lucasfagundes.android_pokedex.view
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import androidx.recyclerview.widget.LinearLayoutManager
-import com.lucasfagundes.android_pokedex.R
 import com.lucasfagundes.android_pokedex.databinding.ActivityMainBinding
 import com.lucasfagundes.android_pokedex.domain.Pokemon
+import com.lucasfagundes.android_pokedex.domain.PokemonType
 
-private var _binding: ActivityMainBinding? = null
-private val binding: ActivityMainBinding get() = _binding!!
+private lateinit var binding: ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
 
         val recyclerView = binding.pokemonRecyclerView
 
-        val layoutManager = LinearLayoutManager(this)
-        recyclerView.layoutManager = layoutManager
-
+        val charmander = Pokemon(
+            "https://assets.pokemon.com/assets/cms2/img/pokedex/detail/214.png",
+            4,
+            "Charmander",
+            listOf(PokemonType("Fire"))
+        )
```

Comment:
acredito que vc esteja criando esse objeto aqui somente pra teste ne, mas mesmo assim sugiro que de nomes as variaveis
```suggestion
        val charmander = Pokemon(
            image = "https://assets.pokemon.com/assets/cms2/img/pokedex/detail/214.png",
            number = 4,
            name = "Charmander",
            typeList = listOf(PokemonType("Fire"))
        )
```